### DISCUSSION TITLE
Export suboptimal solutions?

### DISCUSSION THREAD

**Discussion by jbf302** [coin-or/pulp#607] [Q&A]
tl;dr - is it possible to export all of the _sub_optimal solutions from PuLP for analysis?

I run a short script after every NFL game that imports the latest results and uses them as an input to my simple Pulp optimization function, and that tells me the "highest possible" score I or any other entrant could possibly achieve. Without getting too deep into the details of the contest, the script I run works great!

To further my analysis and see who had the "best" entry in the contest, I'd like see a further breakdown of the **non-optimal** solutions, as well. What was the mean score of these suboptimal entries? What was a key decision that made a difference in the entrant's final score?

Is it possible to export all of the _sub_optimal solutions for analysis? Maybe as a dataframe?
